### ISSUE TITLE
other player floating a considerably far distance away from the vehicle they are in when driving mode is activated (comes with image)

### ISSUE THREAD

**Issue by sock994** [MSCMP/MSCMP#23]
![20180626184635_1](https://user-images.githubusercontent.com/40743904/42138787-ce3cdac4-7d7a-11e8-9762-9c1c788abcc0.jpg)

**Comment by DarkKostas** [comment]
This is happening because on your screen the car was in a different position when he entered the car and after that there's no "player" movement synchronise, but only vehicle as the player is being attached on the car. It's known bug and was introduced after the new optimised loading. Will be fixed once the initial vehicle positions are synchronised again, as now they are only once you enter them. @RootKiller

**Comment by CurtisVL** [comment]
This issue might have been resolved in 0.3 by the new idle object sync. If not, please re-open a new issue.
The latest builds can be downloaded from our Discord server. :)
